### PR TITLE
Add a Vagrantfile

### PR DESCRIPTION
**Pull request by stefwalter**
Add a vagrant file so people can easily test out Cockpit, including loading assets from the pkg/ directory directly.

### CODE REVIEW

**Comment by stefwalter** [comment]
This depends on #2885 ... and then getting #2885 into a COPR release.

**Comment by stefwalter** [comment]
This also needs documentation.

**Comment by mvollmer** [comment]
What is `tools/vagrant/.gitignore` for?

**Comment by stefwalter** [comment]
> What is tools/vagrant/.gitignore for?

I'd like to put files in this directory to make available inside the VM. And .gitignore is a way to have git keep an empty directory around.

**Comment by stefwalter** [comment]
@mvollmer For vagrant, we're going to need a way to fail loading Cockpit if cockpit-bridge and cockit-ws aren't new enough (since those are static in the Vagrant, and the user needs to ```vagrant destroy && vagrant up``` once they get out of date).

So I need to figure out how to accomplish that. I wonder if there are other similar use cases.

**Comment by stefwalter** [comment]
Blocked until a COPR release.

**Comment by stefwalter** [comment]
@kybaker After the Cockpit release next week, we should have a working Vagrant going.

**Comment by stefwalter** [comment]
Although this won't work completely perfectly before 0.79 is released and in COPR, we need to get this in anyway so it's part of that release.

**Comment by stefwalter** [comment]
Test failure is #2914

**Comment by stefwalter** [comment]
TestKubernetes failure is #2918

**Comment by stefwalter** [comment]
Travis failure is Travis network connection problem.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        ln -snf /cockpit/pkg /usr/local/share/cockpit
+        ln -snf /cockpit/pkg /root/.local/share/cockpit
+        ln -snf /cockpit/pkg /home/admin/.local/share/cockpit
+
+        dnf copr enable -y sgallagh/cockpit-preview
+        dnf update -y docker
+        dnf install -y kubernetes atomic subscription-manager etcd pcp realmd NetworkManager \
+                storaged storaged-lvm2 git yum-utils
+        dnf install -y cockpit cockpit-pcp
+	debuginfo-install -y cockpit cockpit-pcp
```
Comment (dperpeet) [inline Vagrantfile]: indentation

**Pull request closed without merging**